mantidproject/mantid
# DNS powder TOF GUI functionality improvement

**Description of work.**

This PR improves functionality of the Powder TOF mode of the DNS Reduction GUI. In particular: 

- The drop-down combo box in the upper left corner of the "Data" tab is replaced by two separate "Sample Data" and "Standard Data" buttons
- As soon as the path for standard data is specified by the user in the "Standard Files Directory" field of the "Paths" tab, the standard data are loaded and selected for reduction by default
- The obsolete "Read All" button is removed from the interface
- Documentation is re-structured and updated

**To test:**

- Download test data: [TOF_Powder_Vanadium.zip](https://github.com/mantidproject/mantid/files/8629427/TOF_Powder_Vanadium.zip)
- Open the reduction GUI [Interfaces->Direct->DNS Reduction]
- Browse to the directory where the test data are located using the "Paths" tab of the GUI
- Switch to the "Data" tab. The expected default behavior is: 
  - Sample test data are loaded and available for selection in "Sample Data" view.
  - Standard test data are loaded and selected for reduction in "Standard Data" view. Switch to the "Standard Data" to make sure that the default setting works.  
- Open the "Options" tab to select options for data reduction 
- Reduce the data and generate a data reduction script using the "Script Generator" tab
- Visualize results using the "Plotting" tab
- Make sure that the updated documentation is consistent with the implemented modifications

Fixes #34828. 





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
